contracts/Request.sol

```diff
@@ -1,4 +1,4 @@
-pragma solidity ^0.5.0;
+pragma solidity >=0.5.3 <0.7.0;
```

Comment:
```suggestion
pragma solidity 0.7.0;
```

Reply:
This contract is used (inherited) by the truffle box, so we could allow a range of solidity versions. :)